inspire_utils/date.py

```diff
@@ -189,7 +189,7 @@ def pprint(self):
 
         """
         if not self.month:
-            return dates.format_date(datetime.date(self.year, 1, 1), 'yyyy', locale='en')
+            return dates.format_date(datetime.date(self.year, 1, 1), 'YYYY', locale='en')
```

Comment:
that's incorrect, you have to fix the two other `YYYY` -> `yyyy`

Reply:
Sorry, stupid mistake 😓, thanks for pointing it out.